index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Terri's CSS Tutorials 1-5 Github Page</title>
+  </head>
+  <body>
+    <strong>This Github Page reflects:</strong>
+    <ul>
+      <li>
+        my work through the first five tutorials of Zastrow's How to Style HTML
+        with CSS.
+      </li>
+      <li>my application of that knowledge while flexing my Github Skills.</li>
+    </ul>
+
+    <strong>Here are links to my implementations of the tutorial concepts:</strong>
+     <ol>
+      <li>
+        <a href="/stylewithcss/1apply-css/index.html"><em>How To Apply CSS Styles to HTML with Cascade and Specificity</em></a> 
+      </li>
+      <li>
+        <a href="/stylewithcss/2select-html/index.html"><em>How To Select HTML Elements to Style with CSS</em></a>
+      </li>
+      <li>
+        <em>How To Style Text Elements with Font, Size, and Color in CSS</em>
+      </li>
+      <li>
+        <em>How To Work with the Box Model in CSS</em>
+      </li>
+      <li>
+        <em>How to Use Common Units in CSS</em>
```

Comment:
If all of these are `em`phasized, then none of them are. Make sure you're using `em` for elements that have a logical reason for emphasis and not for style.